### ISSUE TITLE
Incompatibility with the attr-encrypted gem

### ISSUE THREAD

**Issue by theharq** [thoughtbot/clearance#685]
Hi! I don't know if I should report this issue in the `attr-encrypted` project or maybe here is a good place.. in any case, there is an issue when we include the  `Clearance::User` module.

When we include this module the `PasswordStrategies::BCrypt` is included too by default, which adds a private method `encrypt` with one parameter:

 > ```
def encrypt(password)
   ::BCrypt::Password.create(password, cost: cost)
 end
```
https://github.com/thoughtbot/clearance/blob/master/lib/clearance/password_strategies/bcrypt.rb#L29

The `attr-encrypted` gem defines also a `encrypt` method but with 2 parameters which is called from this line:
> `send("#{encrypted_attribute_name}=", encrypt(attribute, value))`
https://github.com/attr-encrypted/attr_encrypted/blob/master/lib/attr_encrypted.rb#L159

Now if we try to update the object, we are going to see this error:

```
user.update(access_token: "test")
ArgumentError: wrong number of arguments (given 2, expected 1)
```

Thank you for your efforts and hard work maintaining this gem 😄

**Comment by tadp** [comment]
Steps to reproduce:

1. add `gem 'attr_encrypted'` to the Gemfile (notice the underscore)
2. add 
```   include Clearance::User
        extend AttrEncrypted
        attr_encrypted :token``` to your user.rb
3. `user.update(token: 'test')` (as above)

Notice this error:
```
ArgumentError: wrong number of arguments (given 2, expected 1)
from /Users/USER/.rvm/gems/ruby-2.3.1/gems/clearance-1.14.1/lib/clearance/password_strategies/bcrypt.rb:29:in `encrypt'
```

**Comment by derekprior** [comment]
Closed by #693
